docs/deploy.md

```diff
@@ -209,8 +209,29 @@ truffle(zen)> withdrawalContract.submitWithdrawalRequest(mcAddress, { value: 100
 
 **Note:** The mcAddress needs to be defined or replaced by the string containing the Horizen mainchain address. 
 
-The target mainchain address of a backward transfer has to be a valid address: starting with “**zn**” for *mainnet* or “**zt**” for *testnet*.
+The target mainchain address of a backward transfer must be a **valid address** with the following format:
 
+1. Use **zn** for *mainnet* or **zt** for *testnet*.
+
+
+    **Example:** ztWBHD2Eo6uRLN6xAYxj8mhmSPbUYrvMPwt
```

Comment:
I don't think the `Example` lines are being formatted correctly on the site https://ramone-tools-1912.evm-documentation.pages.dev/docs/deploy/#interact-with-native-smart-instance

Reply:
made the adjustment to the Example lines